exchanges/wshandler/websocket_connection_types.go

```diff
@@ -0,0 +1,26 @@
+package wshandler
+
+import (
+	"sync"
+	"time"
+
+	"github.com/gorilla/websocket"
+)
+
+// WebsocketConnection contains all the datas needed to send a message to a WS
+type WebsocketConnection struct {
+	sync.Mutex
+	Verbose      bool
+	RateLimit    float64
+	ExchangeName string
+	URL          string
+	ProxyURL     string
+	Wg           sync.WaitGroup
+	Connection   *websocket.Conn
+	Shutdown     chan struct{}
+	// These are the request IDs and the corresponding response JSON
+	IDResponses map[int64][]byte
+}
+
+var noResponseFoundTimeout = 20 * time.Millisecond
```

Comment:
It would be good to make these exportable so that they can become configurable during exchange setup as these values would fluctuate between exchanges similar to how each exchange has their own rate limiter/requester timeout vals now